### ISSUE TITLE
Review recent translations

### ISSUE THREAD

**Issue by nijel** [WeblateOrg/weblate#2468]
Hi Michal, Thanks for you answer.
I had a look to the proposed link but it's still not what I request / need to check translation last activity ;-(.

I request a list / grid displaying last changes made to a component with:
- date (sorted)
- source string
- raw string = last translation value (without any change highlighted)
- type of event (modification or first translation)
- the user who performed the change
- a link to string (to perform a correction).
these infos and behavior will allow a weblate translator to perform very fast checks on last translations!

It's quite like a "review" but this list:
- should be available immediately (in 1 click without filling any form)
- and should display several translations strings for one language on the same page / at the same time to prevent wasting time to click on next and to give a quick "overview".

To sum up, it's still my initial request 1 year 1/2 ago: being able to display last translated strings in a view (new display mode or new view) like https://hosted.weblate.org/projects/weblate/master/fr/#history .

Can you please consider my request by re-opening this issue?

_Originally posted by @ldmpub in https://github.com/WeblateOrg/weblate/issues/1570#issuecomment-449560396_

**Comment by ldmpub** [comment]
Hi Michal,

Happy new year!
Thanks to have reopened this issue. I'm trying to "push" this improvement since July 2017, see for instance #1569.

**Comment by nijel** [comment]
@ldmpub  Is there still anything missing in the history view to achieve this? To me, it seems that we now have all the bits there...

**Comment by ldmpub** [comment]
Hi Michal,

It's always a pleasure to deal with you, even old issues are still "active" ;-).

I agree with you, improvements made to the  history view (like https://hosted.weblate.org/projects/weblate/application/fr/#history) are in the good direction, now displaying:
- author
- source string
...
everything listed in my first post is now available except what I have called the "**raw translation string**" = last true value of the translation (without any change highlighted ie no red / green color but the "full sentence user will see in the app").

I suggest:
- option 1: to add a toggle in the current view (we can call it "highlight changes in translations strings" or the opposite "display final / raw translation strings")
- option 2: add a new view with all information needed and call this new view "Simplified history".

Note: advanced search is an alternative (filtered on "changed date") but it's not fast to use because "reviewer" needs to navigate between each string.

I'm ready to discuss my needs and alternatives,

**Comment by ldmpub** [comment]
option 3 can be to add / display the "raw translation" in a tooltip when over the "highlighted translation".

**Comment by nijel** [comment]
I do review old ones sometimes to see if they weren't fixed by some other changes...

The search might help here as well: https://hosted.weblate.org/search/weblate/?offset=&q=+changed%3A%3E%3D2020-05-01+AND+language%3Afr&sort_by=-priority%2Cposition&checksum=

**Comment by ldmpub** [comment]
You're right that's a solution I've suggested above "Note: advanced search is an alternative (...)".

**1. Two different layouts for search results**
Your search is done at project level and layout displays several translations on the same page => nice, sample https://hosted.weblate.org/search/weblate/?offset=&q=+changed%3A%3E%3D2020-05-01+AND+language%3Afr&sort_by=-priority%2Cposition&checksum=
The search at the "component" level I'm using has a "mono-translation layout" => user should navigate between strings, see https://hosted.weblate.org/translate/weblate/website/fr/?offset=&q=+changed%3A%3E%3D2020-05-01+&sort_by=-priority%2Cposition&checksum= 

**2. Missing language criteria in search form**
The current search form is not designed to add a language criteria and this useful criteria is not is the "queries samples" section.

I suggest:

- add somewhere in the UI a link to your predefined search query (at Project and Component levels).

or 

- in the search form:
-- add an option "display all translations strings found on the same page" or the contrary "display 1 translation string found per page"
-- add a language criteria "target language" (should be filled with current translation language by default).

Thanks for our discussion.

**Comment by nijel** [comment]
See https://github.com/WeblateOrg/weblate/issues/3063 for improvements of the search UI.

**Comment by nijel** [comment]
> Two different layouts for search results

Let's deal with this one in https://github.com/WeblateOrg/weblate/issues/1031

> add somewhere in the UI a link to your predefined search query (at Project and Component levels).

This is covered by https://github.com/WeblateOrg/weblate/issues/3060

> add an option "display all translations strings found on the same page" or the contrary "display 1 translation string found per page"

You can switch to zen mode by one click when on the results page.

> add a language criteria "target language" 

Added now.
